Makefile

```diff
@@ -13,7 +13,7 @@ endif
 .FORCE:
 
 all: clean
-	babel src -d dist --source-maps
+	npm run build
```

Comment:
Can we run the command directly from the makefile rather than from npm scripts?  The PATH environment variable will already be set from the Makefile, so you can run project-scoped npm binaries from Makefile without having to use npm scripts

Reply:
Sure will do that